discord/interactions.py

```diff
@@ -164,12 +173,16 @@ def _from_data(self, data: InteractionPayload):
                 # The fallback to Object for guild causes a type check error but is explicitly allowed here
                 self.user = Member(state=self._state, guild=guild, data=member)  # type: ignore
                 self._permissions = self.user._permissions or 0
+
+            self.guild_locale = try_enum(Locale, data.get('guild_locale', 'en-US'))
```

Comment:
It might be better to move this up and use try except instead of storing it here.

Reply:
Fixed, I... think. Unless I misunderstood instructions >_<